pandas/tests/extension/test_decimal.py

```diff
@@ -177,7 +177,8 @@ class TestMissing(base.BaseMissingTests):
 
 
 class TestMethods(base.BaseMethodsTests):
-    @pytest.mark.xfail(reason="NaN Sorting")
+    @pytest.mark.parametrize('dropna', [True, False])
+    @pytest.mark.xfail(reason="value_counts not implemented yet.")
```

Comment:
It is actually already working for me in case there are no NaNs:

```
In [13]: d = DecimalArray([decimal.Decimal(1.0), decimal.Decimal(1.0), decimal.Decimal('1.3')])

In [14]: pd.value_counts(d)
Out[14]: 
1      2
1.3    1
dtype: int64
```

Maybe we should test that already?